### PR TITLE
Add FindTagCommand 

### LINKED ISSUE #88: Add find tag command

### PR DESCRIPTION
**Pull request by zienxu**
Add Find Tag Command feature to search for tags
closes #88

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/FindTagCommand.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    private final TagContainsKeywordsPredicate predicate;
+
+    public FindTagCommand(TagContainsKeywordsPredicate predicate) {
+        this.predicate = predicate;
+    }
+
+    @Override
+    public CommandResult execute(Model model) {
+        logger.info("Executing FindTagCommand...");
```
Comment (achl1012) [inline src/main/java/seedu/address/logic/commands/FindTagCommand.java:34]: Good use of logging!

File: src/main/java/seedu/address/logic/parser/FindTagCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+     * Parses the given {@code String} of arguments in the context of the FindTagCommand
+     * and returns a FindTagCommand object for execution.
+     *
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public FindTagCommand parse(String args) throws ParseException {
+        String trimmedArgs = args.trim();
+        if (trimmedArgs.isEmpty()) {
+            throw new ParseException(
+                    String.format(Messages.MESSAGE_INVALID_COMMAND_FORMAT, FindTagCommand.MESSAGE_USAGE));
```
Comment (kimberlytmq) [inline src/main/java/seedu/address/logic/parser/FindTagCommandParser.java:25]: can consider creating custom invalid command message for filter command?

File: src/main/java/seedu/address/logic/commands/FindTagCommand.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import java.util.logging.Logger;
+
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.logic.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.tag.TagContainsKeywordsPredicate;
+
+
+
+/**
```
Comment (kimberlytmq) [inline src/main/java/seedu/address/logic/commands/FindTagCommand.java:12]: good use of javadoc comments!

**Review by kimberlytmq — DISMISSED** [review]
lgtm!

File: src/main/java/seedu/address/logic/parser/FindTagCommandParser.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+/**
+ * Parses input arguments and creates a new {@code FindTagCommand} object.
+ */
+public class FindTagCommandParser implements Parser<FindTagCommand> {
+    /**
+     * Parses the given {@code String} of arguments in the context of the FindTagCommand
+     * and returns a FindTagCommand object for execution.
+     *
+     * @throws ParseException if the user input does not conform the expected format
+     */
```
Comment (achl1012) [inline src/main/java/seedu/address/logic/parser/FindTagCommandParser.java:20]: Good use of javadoc comments!

**Review by achl1012 — DISMISSED** [review]
LGTM! 👍

**Review by SlothyCat — APPROVED** [review]
LGTM!

**Review by SlothyCat — APPROVED** [review]
LGTM! 😄

**Review by achl1012 — APPROVED** [review]
LGTM!

**Pull request merged**